http/file_server.ts

```diff
@@ -155,6 +155,12 @@ export interface ServeFileOptions {
    * `filePath`.
    */
   fileInfo?: Deno.FileInfo;
+
+  /** Headers to add to each response
+   *
+   * @default {[]}
+   */
+  headers?: string[];
```

Comment:
I think this should be a [`Headers`](https://developer.mozilla.org/en-US/docs/Web/API/Headers) interface. I also think the same can be said about `serveDir`.